packages/autocannon-ui-frontend/src/components/CompareDialog.jsx

```diff
@@ -1,43 +1,53 @@
 import React, { useState, useEffect, useMemo } from 'react'
 import T from 'prop-types'
+
 import {
-  IconButton,
   Dialog,
   DialogTitle,
   DialogContent,
   CircularProgress,
-  makeStyles,
   Grid,
+  IconButton,
   TableContainer,
   Paper,
   Table,
   TableHead,
   TableRow,
   TableCell,
   TableBody
-} from '@material-ui/core'
-import CloseIcon from '@material-ui/icons/Close'
-import { Alert } from '@material-ui/lab'
+} from '@mui/material'
+import CloseIcon from '@mui/icons-material/Close'
+import Alert from '@mui/material/Alert'
+import { styled } from '@mui/material/styles'
+
+const PREFIX = 'CompareDialog'
+const classes = {
+  dialog: `${PREFIX}-dialog`,
+  dialogTitle: `${PREFIX}-dialogTitle`,
+  dialogContent: `${PREFIX}-dialogContent`,
+  closeButton: `${PREFIX}-closeButton`,
+  statHeader: `${PREFIX}-statHeader`
+}
 
-const useStyles = makeStyles({
-  dialog: {
-    margin: '0 20px'
+const Root = styled('div')(({ theme }) => ({
+  [`& .${classes.dialog}`]: {
+    margin: `0 ${theme.spacing(2.5)}`
   },
-  dialogTitle: {
-    paddingRight: '48px'
+  [`& .${classes.dialogTitle}`]: {
+    paddingRight: theme.spacing(6)
```

Comment:
why this new way of defining styles?

Reply:
We do need if if we want to use the chrome-aws-lambda package. I am checking if we can remove it or not

Reply:
I followed this since we are using emotion:
https://mui.com/material-ui/migration/migrating-from-jss/

I think material ui will move away from makeStyles (not now, maybe not in the near future either but it will happen.

Reply:
my thoughts as well :)